tests/test_benchmark.py

```diff
@@ -229,6 +229,86 @@ def benchmark(
     )
 
 
+def benchmark_local(
+    label,
+    masses,
+    x0,
+    v0,
+    box,
+    bound_potentials,
+    ligand_idxs,
+    hmr=False,
+    verbose=True,
+    num_batches=100,
+    steps_per_batch=1000,
+):
+    """
+    TODO: configuration blob containing num_batches, steps_per_batch, and any other options
+    """
+
+    seed = 1234
+    dt = 1.5e-3
+    temperature = constants.DEFAULT_TEMP
+    friction = 1.0
+    seconds_per_day = 86400
```

Comment:
nit: style preference for `24 * 60 * 60`

Reply:
Done in [7c8d6f1](https://github.com/proteneer/timemachine/pull/968/commits/7c8d6f161b6b3545df69eff2c85a3d78ce4bb982)